### ISSUE TITLE
bug: wasm-opt path is incorrect, extra `bin` folder in the path

### ISSUE THREAD

**Issue by HarikrishnanBalagopal** [rustwasm/wasm-pack#1263]
# Bug

Can no longer build with `wasm-pack build` because the path to the `wasm-opt` binary is incorrect.
Started after updating from `0.10.3` to `0.11.0` using the command given here https://rustwasm.github.io/wasm-pack/installer/
```console
$ curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

## Steps to reproduce

1. Run `wasm-pack build` in a Cargo Rust project

## Expected behaviour

Build should succeed like it did in `0.10.3` version.

## Actual behaviour

Build failure because it adds an extra `bin` folder in the path to the `wasm-opt` binary.
```console
$ wasm-pack build
[INFO]: 🎯  Checking for the Wasm target...
[INFO]: 🌀  Compiling to Wasm...
    Finished release [optimized] target(s) in 0.08s
[INFO]: ⬇️  Installing wasm-bindgen...
[INFO]: found wasm-opt at "/usr/local/bin/wasm-opt"
Error: /usr/local/bin/bin/wasm-opt binary does not exist
To disable `wasm-opt`, add `wasm-opt = false` to your package metadata in your `Cargo.toml`.
Caused by: /usr/local/bin/bin/wasm-opt binary does not exist
To disable `wasm-opt`, add `wasm-opt = false` to your package metadata in your `Cargo.toml`.
```
## Context

Bug happens on version
```console
$ wasm-pack --version
wasm-pack 0.11.0
```

## Relevant code
https://github.com/rustwasm/wasm-pack/blob/d5cb84e3e802293154a1b85db310cbb458b5a941/src/wasm_opt.rs#L26
https://github.com/rustwasm/wasm-pack/blob/d5cb84e3e802293154a1b85db310cbb458b5a941/src/wasm_opt.rs#L55-L58

Change made in this PR:
https://github.com/rustwasm/wasm-pack/commit/30f2b0a26232b54c8ab7e2e09751170f52856eab#diff-d9cd988699c3224bc78800edb64a7cb60d3cae9833460f79ce2d586b9555cd83L26-R26

**Comment by drehren** [comment]
This just happened to me as well, but to trigger it, I had to first `cargo install wasm-opt`.

**Comment by nekopsykose** [comment]
this is fixed in 0.11.1
